src/metrics/blocks.ml

```diff
@@ -0,0 +1,14 @@
+open Prometheus
+open Config
+
+let blocks_produced_total =
+  let help = "Total number of blocks produced" in
+  Counter.v ~help ~namespace ~subsystem "blocks_produced_total"
+
+let blocks_signed_total =
+  let help = "Total number of blocks signed" in
+  Counter.v ~help ~namespace ~subsystem "blocks_signed_total"
+
```

Comment:
I think it'd be useful to also have total transactions received and included, so we would be able to measure throughput and delay for a transaction to be confirmed.

Reply:
The goal of this PR is to get the bare minimum to just have dashboards. Please add this to upcoming PRs :smile:
Would also love it if @gsebil08 and @rosalogia could add some metrics to the pollinate library that we can expose via deku.

Reply:
Sounds like a great idea :smile: